apps/climatemappedafrica/src/payload/fields/HURUMapURL/index.js

```diff
@@ -0,0 +1,109 @@
+import { Button } from "payload/components/elements";
+import {
+  Label,
+  TextInput,
+  useField,
+  useFormFields,
+} from "payload/components/forms";
+import { createElement, useState, useEffect } from "react";
+
+// TODO: @kelvinkipruto Handle i18n
+function HURUMapURL(props) {
```

Comment:
```suggestion
function HURUmapURL(props) {
```
It's [`HURUmap`](https://hurumap.org/) and not `HURUMap`